application/borrower/model.py

```diff
@@ -0,0 +1,18 @@
+from application import db
+
+
+class Borrower(db.Model):
+    __tablename__ = 'borrower'
+
+    id = db.Column(db.Integer, primary_key=True)
+    forename = db.Column(db.String, nullable=False)
+    middlename = db.Column(db.String, nullable=True)
+    surname = db.Column(db.String, nullable=False)
+    dob = db.Column(db.String, nullable=False)
+    gender = db.Column(db.String, nullable=True)
+    phonenumber = db.Column(db.String, nullable=False)
+    address = db.Column(db.String, nullable=False)
+
+    def save(self):  # pragma: no cover
```

Comment:
This comment is meaningless to someone reading it for the first time, why is it here?